### PR TITLE
B-18990 Add shipment locator officer pages KICKBACK 2 MAIN

### PR DESCRIPTION
**Pull request by KonstanceH**
## [B-18990](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A909786)

Integration KICKBACK 2 PR: https://github.com/transcom/mymove/pull/12537
Integration KICKBACK PR: https://github.com/transcom/mymove/pull/12524
Integration PR: https://github.com/transcom/mymove/pull/12496

## Summary
This PR adds shipment locator to all shipment cards from all officer views.
Also removes extra heading space and formatting for tags like 'PACKET READY FOR DOWNLOAD', 'DIVERSION', etc.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### How to test

1. Create any move or use any existing move
2. Login as any officer(TXO, SC)
3. Confirm the shipment locator is under the Shipment type. (Similar to the moveLocator for the customer)

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

## Screenshots
Screenshot 2024-04-18 at 1 29 01 PM
Screenshot 2024-04-18 at 1 26 14 PM
Screenshot 2024-04-17 at 12 48 07 PM

### CODE REVIEW

File: src/components/Office/ShipmentDisplay/ShipmentDisplay.module.scss [hunk 1]
```diff
@@ -11,15 +11,25 @@
...
+        }
 
+        h5 {
+          font-weight: normal;
+          @include u-margin(0);
+          @include u-margin-x(1);
+          @include u-text('gray-cool-50', 'normal');
+          @include u-margin-top(0);
+        }
+      }
```
[the remark this answers is not in the document]
Comment (KonstanceH) [reply src/components/Office/ShipmentDisplay/ShipmentDisplay.module.scss]: Okay, I got rid of the indent and added spaces

File: src/components/Office/ShipmentDisplay/ShipmentDisplay.jsx
Comment (taeJungCaci) [inline src/components/Office/ShipmentDisplay/ShipmentDisplay.jsx:1]: Only thing I see missing is the `shipmentLocator: PropTypes.string,` in this INT PR: https://github.com/transcom/mymove/pull/12524/files#diff-a4a865709ef18438fcea3a7cdb9a0bf62e4cd3612b9ccd9cbf58389ea8f8b66a

Comment (KonstanceH) [reply src/components/Office/ShipmentDisplay/ShipmentDisplay.jsx:1]: okay just added it in

**Review by taeJungCaci — APPROVED** [review]
LGTM!

**Comment by KonstanceH** [comment]
> [diff_output.txt](https://github.com/transcom/mymove/files/15275945/diff_output.txt)
> 
> See diff output above. It shows diffs upon individual comparison but I believe each of the changes covers each other and that there aren't any diffs on the whole. Please verify.
> 
> 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽 👽

Yea I agree everything looks in the right place to me

**Review by JamesHawks224 — APPROVED** [review]
lgtm

**Pull request merged**